<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingSetupDesign</class>
 <widget class="QWidget" name="TestingSetupDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestingSetupDesign</string>
  </property>
  <widget class="QComboBox" name="patient_select">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="testing_select">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Choix du test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de stimuli</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>560</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>DÃ©marrer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Valeurs</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Consigne :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="consigne">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>consigne_text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Valeurs valides</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="all_values">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valid_values">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
